api/src/main/java/org/apache/cloudstack/api/command/user/offering/ListDiskOfferingsCmd.java

```diff
@@ -44,6 +45,12 @@ public class ListDiskOfferingsCmd extends BaseListDomainResourcesCmd {
     @Parameter(name = ApiConstants.NAME, type = CommandType.STRING, description = "name of the disk offering")
     private String diskOfferingName;
 
+    @Parameter(name = ApiConstants.ACCOUNT, type = CommandType.STRING, description = "list resources by account.")
```

Comment:
@RodrigoDLopez 
If you use `BaseListProjectAndAccountResourcesCmd`,  `accountName` and `projectId` are not needed any more.